content/guides/internal/ui-elements/tabs.md

```diff
@@ -38,5 +40,7 @@ multiple `Tab` items with titles.
 
 Tabs use cookies to try and remember the user's choice. Please use the standard
 title `cURL`, `Java`, `.NET`, `Python` and `Node` to refer to our currently
-supported languages and the user will automatically be presented with the sample
-in their language of choice.
\ No newline at end of file
+supported languages and the user will automatically be presented with the
+sample in their language of choice.
+
+[code_block_guide]: guide://markdown/code-blocks
```

Comment:
@cbetta I thought this would work, but would appreciate your input on how to cross-link the internal guides

Reply:
It should be the full path from the `content` folder down, so in this case that would be `guide://internal/markdown/code-blocks`